src/fortran/hello.f90

```diff
@@ -1,6 +1,6 @@
-c SPDX-FileCopyrightText: 2020 Intel Corporation
-c
-c SPDX-License-Identifier: MIT
+! SPDX-FileCopyrightText: 2020 Intel Corporation
+!
+! SPDX-License-Identifier: MIT
```

Comment:
Just curious. Why did you change this?

Reply:
There's a "source form" concept in Fortran. Unless the source form is specified through a compiler option, it is determined by the compiler based on file extension: .f90 denotes "free form" while ".f" denotes "fixed form". The 2 forms shall not be mixed in the same program unit. The fixed form is legacy and has restrictions on where a statement can appear within a line. The 2 forms have different approaches to comments. '!' starts the comment in the "free form". If the comment is started with 'c' in an .f90 program compiled w/ default options, there'll be a syntax error.
The examples don't use this file at the moment. This change fixes the syntax error in case they would.

More details can be found in section 3.3.2.3 in F2008 standard at https://j3-fortran.org/doc/year/10/10-007r1.pdf